backend/amd64/proc.ml

```diff
@@ -401,6 +404,7 @@ let max_register_pressure =
              | Ioffset_loc (_, _) | Ifloatarithmem (_, _)
              | Ibswap _ | Ifloatsqrtf _ | Isqrtf)
   | Iname_for_debugger _ | Iprobe _ | Iprobe_is_enabled _ | Iopaque
+  | Ibeginregion | Iendregion
```

Comment:
Should it match `destroyed_at_oper` where `Iendregion` has needs an extra reg?

Reply:
Makes sense, changed.